inc/field.class.php

```diff
@@ -919,7 +919,16 @@ public static function showForTab($params)
         }
 
         $html_id = 'plugin_fields_container_' . mt_rand();
-        echo "<div id='{$html_id}'>";
+        if (strpos($current_url, "helpdesk.public.php") !== false) {
+            echo "<div id='{$html_id}' class='card-body row mx-0' style='border-top:0'>";
+            echo "<div class='offset-md-1 col-md-8 col-xxl-6'>";
+            $item->field_options = [
```

Comment:
This will trigger a warning on PHP 8.2+ (see https://php.watch/versions/8.2/dynamic-properties-deprecated).